tests/test_table.py

```diff
@@ -8,14 +10,224 @@
+                    )
+                ),
+            ],
+            create_table(
+                pd.Series(
+                    [1., 2., 3., 4.],
+                    index=audformat.filewise_index(['f1', 'f2', 'f3', 'f4']),
+                )
+            ),
+        ),
+        pytest.param(  # value mismatch
+            [
+                create_table(
+                    pd.Series(
+                        [1.],
+                        index=audformat.filewise_index('f1'),
+                    )
+                ),
+                create_table(
+                    pd.Series(
+                        [-1.],
+                        index=audformat.filewise_index('f1'),
+                    )
+                ),
+            ],
+            None,
+            marks=pytest.mark.xfail(raises=ValueError),
+        ),
+        # different columns
+        (
+            [
+                create_table(
+                    pd.Series(
+                        [1., 2.],
+                        index=audformat.filewise_index(['f1', 'f2']),
+                        name='c1',
+                    )
+                ),
+                create_table(
+                    pd.Series(
+                        [2., 3.],
+                        index=audformat.filewise_index(['f2', 'f3']),
+                        name='c2',
+                    )
+                ),
+            ],
+            create_table(
+                pd.DataFrame(
+                    {
+                        'c1': [1., 2., np.nan],
+                        'c2': [np.nan, 2., 3.],
+                    },
+                    index=audformat.filewise_index(['f1', 'f2', 'f3']),
+                )
+            ),
+        ),
+        # filewise + segmented
+        (
+            [
+                create_table(
+                    pd.Series(
+                        [1.],
+                        index=audformat.filewise_index('f1'),
+                    )
+                ),
+                create_table(
+                    pd.Series(
+                        [1.],
+                        index=audformat.segmented_index('f1', 0, 1),
+                    )
+                ),
+            ],
+            create_table(
+                pd.Series(
+                    [1., 1.],
+                    index=audformat.segmented_index(
+                        ['f1', 'f1'],
+                        [0, 0],
+                        [None, 1],
+                    ),
+                )
+            ),
+        ),
+        (
+            [
+                pytest.DB['files'],
+                pytest.DB['segments'],
+            ],
+            create_table(
+                audformat.utils.concat(
+                    [
+                        pytest.DB['files'].df,
+                        pytest.DB['segments'].df,
+                    ]
+                )
+            )
+        )
+    ]
+)
+def test_add(tables, expected):
```

Comment:
I think we are missing tests for adding an empty table to a table with content and the other way around.

Also, all the tests above check what is happing if you do `a + b`. Can we assume that `a + b + c` will then automatically work as well?

Reply:
I added the proposed tests.

Reply:
The `a + b + c` test is now only for empty tables, would be nice to also have one for tables with content.

Reply:
It's here: https://github.com/audeering/audformat/pull/49/files#diff-956061cd60efd54943151485eaa77ab422337bf1298e1f6094f118cd138b7c04R153

Reply:
Sorry, missed this. Thanks.